coding/browser-support.md

```diff
@@ -0,0 +1,30 @@
+# Browser support
+
+Our baseline browser support for our websites. There are two levels of support. 
+
+## Full support
+
+Any browser with full support needs to be pixel perfect and all functionality 
+should work.
+
+| Browser | Version |
+| ------- | ------- |
+| Chrome | 55 > |
+| Firefox | 52 > |
+| Safari | 11 > |
+| IE | 11 > |
+| Edge | 15 > |
+| Opera | 48 > |
```

Comment:
This reads "less than 48", and the lines above are the same.

It should be `>= 48` but it could read better as a literal `48 or higher`.